src/scripts/check-for-new-lunie-versions.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * this script polls the Lunie site every minute and extracts its etag to check if there has been
+ * any new Lunie versions deployed on Netlify while the user had the app open on the browser
+ *
+ * Usage:
+ *
+ * node tasks/poll-index.js
+ */
+
+const fetch = require("node-fetch")
```

Comment:
```suggestion
```

Reply:
Yeah, you are right. This was because I was testing at the beginning with node 🤦‍♀